core/src/main/java/org/springframework/security/authorization/ObservationAuthorizationManager.java

```diff
@@ -69,16 +69,16 @@ public ObservationAuthorizationManager(ObservationRegistry registry, Authorizati
 	}
 
 	@Override
-	public AuthorizationDecision check(Supplier<Authentication> authentication, T object) {
```

Comment:
I believe once you change to `authorize` being the default method, you will be fine to leave implementations as they are.